libcontainer/nsenter/nsexec.c

```diff
@@ -61,7 +63,13 @@ struct clone_t {
 };
 
 struct nlconfig_t {
+	/*
+	 * Stores a pointer to the netlink data payload. All other pointers
+	 * are inside this block. Free it with nl_free().
+	 */
```

Comment:
@cyphar I want to know that why we need to store the pointer to the netlink data payload.  After we call `nl_parse` function, we will never use the payload data. Whether can we free the pointer(allocated by malloc in the function `nl_parse`) at the end `nl_parse` and delete the funciton `nl_free`?